data/saas/config/amplitude_config.yml

```diff
@@ -0,0 +1,67 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Amplitude
+  type: amplitude
+  description: A sample schema representing the Amplitude connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: amplitude.com
+    - name: authorization_key
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+        strategy: api_key
+        configuration:
+          headers:
+            - name: Authorization
+              value: Basic <authorization_key>
```

Comment:
The docs mention `api_key` and `secret_key`, let's update this to use those instead. Make sure to update the Postman collection and the `SaaS Connector Secrets` entry for Amplitude with these values.

Reply:
Changes the header authorisation to basic authentication with api_key as username and secret key as password. And also updated saas secrets and in 1password